src/app/login-button/login-button.component.ts

```diff
@@ -8,15 +8,10 @@ import { AuthService } from '../core/auth.service';
   templateUrl: './login-button.component.html',
   styleUrls: ['./login-button.component.scss']
 })
-export class LoginButtonComponent implements OnInit {
+export class LoginButtonComponent {
 
   constructor(public auth: AuthService, private router: Router) { }
 
-  ngOnInit(): void {
-    const loginButton = document.getElementById('login_button');
-    loginButton.style.display = 'block';
-  }
-
   onLoginClick() {
     this.router.navigateByUrl('/user-profile');
   }
```

Comment:
@A-Robitaille was assigned to fix this

Reply:
It was done in the sprint 2 tutorial. Button stuff was being changed anyway.